llvm/include/llvm/SYCLLowerIR/HostPipes.h

```diff
@@ -0,0 +1,31 @@
+//===------- HostPipes.h - get required into about SYCL Host Pipes --------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// The file contains a number of functions to extract corresponding attributes
+// of host pipe variables and save them as a property set for the runtime.
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "llvm/ADT/MapVector.h"
+
+#include <cstdint>
+
+namespace llvm {
+
+class GlobalVariable;
+
+/// Return \c true if the variable @GV is a host pipe variable.
+///
+/// @param GV [in] A variable to test.
+///
+/// @return \c true if the variable is a host pipe variable, \c false
+/// otherwise.
+bool isHostPipeVariable(const GlobalVariable &GV);
```

Comment:
We have [`SYCLUtils.h`](https://github.com/intel/llvm/blob/sycl/llvm/include/llvm/SYCLLowerIR/SYCLUtils.h) header. Perhaps it is better to move this single helper there? Just not to add a whole to file for a single one-liner